maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlFluentCommand.kt

```diff
@@ -182,4 +172,19 @@ data class YamlFluentCommand(
             optional = selector.optional ?: false,
         )
     }
+
+    companion object {
+
+        @Suppress("unused")
+        @JvmStatic
+        @JsonCreator
+        fun parse(stringCommand: String): YamlFluentCommand {
```

Comment:
Nice! We could do the same for `backPress` and `scroll`. Currently, there is a bit awkward syntax for it: `action: back`, `action: scroll`

Reply:
Added support for this